_howdoi/choose-my-spark-distribution.adoc

```diff
@@ -0,0 +1,68 @@
+= How do I choose my own Apache Spark distribution?
+:page-layout: howdoi
+:page-menu_entry: How do I?
+
+When you install radanalytics.io, your project is set up to use a pre-determined distribution of Apache Spark. You can choose your own Apache Spark distribution with a few additional steps:
+
+.Prerequisites
+
+. Follow the instructions under link:/howdoi/install-radanalyticsio[How do I install radanalytics.io?]
```

Comment:
this should start with the prereq then lead into the instructions. similar to what we've done here https://radanalytics.io/howdoi/deploy-a-spark-cluster-cli

"An OpenShift project with the radanalytics.io manifest installed, see the Get Started instructions for more help."